### PR TITLE
Update CX to Customer Experience 

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

This PR will update any reference of `CX` to `Customer Experience` and remove the topic `Customer Experience Community of Practice` and merge into `Customer Experience` 

The ticket can be viewed by searching for the title `Update existing tag on https://digital.gov/topics/cx/ from "cx" to "customer-experience"` within the tracking system 

### Preview

View testing steps below for preview links

### Solution
Updated any old references and changed the `cx` slug into the new `customer-experience` 

### How To Test

1. Vist the new [Customer Experience](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cx-udpate-sandbox/topics/customer-experience/) topic page and check that the URL is no longer `/topics/cx/ and also check [Topics Page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cx-udpate-sandbox/topics/) and check that CX is not longer listed 
2. Review files changed and verify that CX is no longer referenced

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

Tagging supervisor - @rayestrada

### CODE REVIEW

File: content/topics/customer-experience/_index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # This topic lives at
 # https://digital.gov/topics/cx
 
-slug: "cx"
+slug: "customer-experience"
 
 # Topic Title
 title: "Customer Experience"
```
Comment (bonnieAcameron) [inline content/topics/customer-experience/_index.md:8]:
```suggestion
title: "Customer experience"
```

Comment (bonnieAcameron) [reply content/topics/customer-experience/_index.md:8]: Changing to sentence case (style guide)

**Review by bonnieAcameron — CHANGES_REQUESTED** [review]
One styling change, then all good!

**Review by bonnieAcameron — DISMISSED** [review]

**Comment by ToniBonittoGSA** [comment]
@clmedders This accidentally deletes the CX community page; please restore

**Comment by nick-mon1** [comment]
### Fixes

1. removed duplicates from front matter due to original `cx` duplicates
2. updated footer link `https://digital.gov/topics/cx/` in 4 pages to `https://digital.gov/topics/customer-experience/`
3. added `communities/customer-experience.md` 

### Notes

Customer experience is written in plain language but the other topics are in Title Case.

@bonnieAcameron Should we keep this title case for consistency?

Screen Shot 2023-11-17 at 2 22 15 PM

**Review by nick-mon1 — CHANGES_REQUESTED** [review]
@clmedders I made some corrections and fixes to help get this closed, hope you don't mind.

### Fixes

1. removed duplicates from front matter due to original `cx` duplicates
2. updated footer link `https://digital.gov/topics/cx/` in 4 pages to `https://digital.gov/topics/customer-experience/`
3. added `communities/customer-experience.md` 

### Notes

Customer experience is written in plain language but the other topics are in Title Case.

@bonnieAcameron Should we keep this title case for consistency?

Screen Shot 2023-11-17 at 2 22 15 PM

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**